test/DesyncSafeAnalyzer.Test/PropertyAccessAnalyzerTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using DesyncSafeAnalyzer.Analyzers;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Testing;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+using Xunit;
+
+namespace DesyncSafeAnalyzer.Test
+{
+  public sealed class PropertyAccessAnalyzerTests
```

Comment:
Extend this to test for field assignments as well, not just property assignments.